README.md

```
@@ -43,6 +51,15 @@ export function ContactCaptcha() {
 }
 ```
 
+```tsx
+import { component$ } from "@builder.io/qwik";
+import { ReCaptcha } from "@better-captcha/qwik/provider/recaptcha";
+
+export const ContactCaptcha = component$(() => {
+	return <ReCaptcha sitekey="your-site-key" />;
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)